standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1339,7 +1340,8 @@ public void commitTxn(CommitTxnRequest rqst)
             // corresponding open txn event.
             LOG.info("Target txn id is missing for source txn id : " + sourceTxnId +
```

Comment:
This isn't an info level log. We can remove it actually as the exception is being thrown and that would appear any way